### DISCUSSION TITLE
Implement eslint-plugin-no-secrets

### DISCUSSION THREAD

**Discussion by SaadBazaz** [biomejs/biome#3822] [Ideas]
This plugin identifies strings above a certain entropy threshold and can throw an error. 

For example, if the code contains a secret/key/token:
```ts
const key = "123nsdaoisndon123"
```
It can throw an error. Then usually developers use env variables. 

A really important plugin from a security perspective. 
Very useful if combined with precommit hook with husky.
Has saved countless open-source and proprietary projects.

**Comment by arendjr** [comment]
Please add the request to the discussion here: https://github.com/biomejs/biome/discussions/3

Thanks!
